pages/1.13/security/oss/managing-users/external-users/index.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: layout.pug
+navigationTitle: External users
+title: External users
+excerpt: Managing external DC/OS users
+menuWeight: 10
+---
+<!-- The source repository for this topic is https://github.com/dcos/dcos-docs-site -->
+
+An external user is automatically created for the first user who logs in to the DC/OS cluster via the web interface.
+
+# Log in external users via the web interface
+
+1.  Launch the DC/OS web interface.
+
+2.  Log in with your username (Google, GitHub, and Microsoft) and password.
```

Comment:
"Log in with your username (Google, GitHub, and Microsoft) and password" is, I think, confusing. 

I think this probably means to say "Log in through a single sign-on flow through the provider of your choice (Google, GitHub, or Microsoft).